docs/dev/w3c-webdriver-capabilities.md

```diff
@@ -8,7 +8,7 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-Sauce Labs supports and encourages our users to update their code to take advantage of the [W3C WebDriver Protocol](/dev/glossary/#w3c-webdriver-protocol), which is currently the default protocol used by all major browsers, is fully supported in WebdriverIO 6 and higher, Selenium versions 3.11 and higher, and is required for Selenium 4.0. Using the WebDriver protocol on Sauce Labs requires setting specific capabilities in your code.
+Sauce Labs supports and encourages our users to update their code to take advantage of the [W3C WebDriver Protocol](/dev/glossary/#w3c-webdriver-protocol), which is currently the default protocol used by all major browsers, is fully supported in WebdriverIO 6 and higher, Selenium versions 3.11 and higher, Appium 1.6.5 and higher, and is required for Selenium 4.0 and Appium 2.0. Using the WebDriver protocol on Sauce Labs requires setting specific capabilities in your code.
```

Comment:
Hey @wswebcreation -- thanks for this feedback. This particular doc page is NOT specific to Appium or mobile testing. In fact, it was written to address our support of W3C in our Selenium browser tests, so it actually very Selenium/browser heavy/specific. I have made a few minor edits to this page to at least _not exclude_ Appium, but since it is Kim's doc, I didn't want to edit it too aggressively. Kim and I are planning to collaborate in a rewrite of this page to talk about W3C support across various Sauce Labs test implementations, and we will tackle this comment as part of that effort.